target/linux/ipq806x/base-files/etc/board.d/02_network

```diff
@@ -52,6 +52,9 @@ zyxel,nbg6817)
 	ucidef_set_interface_macaddr "lan" "$(macaddr_add $hw_mac_addr 2)"
 	ucidef_set_interface_macaddr "wan" "$(macaddr_add $hw_mac_addr 3)"
 	;;
+8dev,jalapeno)
```

Comment:
when doing alphanumeric ordering you put numbers before letters ... even if the name suggests otherwise.